### PR TITLE
[Wallet] Add Paste button to Search Input

### LINKED ISSUE #4402: [Wallet] Better send experience for non-verified users
### Expected Behavior

Copy button appears if users have an address copied to clipboard
Figma link- https://www.figma.com/file/zt7aTQ5wuXycIwxq5oAmF9/Wallet--Refresh?node-id=6409%3A837

### Current Behavior

Paste button removed in redesign

### PR DESCRIPTION
**Pull request by i1skn**
### Description

Add Paste icon to search input. This way if address is in clipboard - Paste icon would appear and allow user to quickly enter the address. https://www.figma.com/file/zt7aTQ5wuXycIwxq5oAmF9/Wallet--Refresh?node-id=986%3A4151

### Other changes

Unify `PasteAware` logic and make it UI implementation agnostic.

### Tested

iOS Simulator

### Related issues

- Fixes #4402

### Backwards compatibility

Yes

### Demo

https://drive.google.com/file/d/1PlOO2aOq3zrqpu33tOfoxjzszeX5szkj/view?usp=sharing

### CODE REVIEW

File: packages/mobile/src/backup/BackupPhraseContainer.tsx [hunk 1]
```diff
@@ -10,7 +11,9 @@ import { isValidBackupPhrase, isValidSocialBackupPhrase } from 'src/backup/utils
 import { Namespaces, withTranslation } from 'src/i18n'
 import Logger from 'src/utils/Logger'
 
-const PhraseInput = withTextInputPasteAware(TextInput, { top: undefined, right: 12, bottom: 12 })
+const PhraseInput = withPasteAware(
```
Comment (aaronmgdr) [inline packages/mobile/src/backup/BackupPhraseContainer.tsx]: surprised that something that has *PasteAware* in it needs to be wrapped in another function called *withPasteAware*

Comment (jmrossy) [reply packages/mobile/src/backup/BackupPhraseContainer.tsx]: Yeah +1, can you clarify this?

Comment (i1skn) [reply packages/mobile/src/backup/BackupPhraseContainer.tsx]: Yeah, good catch!

**Comment by i1skn** [comment]
@aaronmgdr @jmrossy I have addressed the comment. PTAL

File: packages/react-components/components/WithPasteAware.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    _interval?: number
+    _isMounted = false
+
+    async componentDidMount() {
+      this._isMounted = true
+      AppState.addEventListener('change', this.checkClipboardContents)
+      this._interval = window.setInterval(async () => {
+        await this.checkClipboardContents()
+      }, 1000) // Every 1s
+      await this.checkClipboardContents()
```
Comment (jmrossy) [inline packages/react-components/components/WithPasteAware.tsx:43]: This strategy (which I realize is based partly on my own code) may not be viable anymore for the new iOS versions. 
https://9to5mac.com/2020/06/24/ios-14-clipboard-notifications/

Comment (jmrossy) [reply packages/react-components/components/WithPasteAware.tsx:43]: Given that this isn't really new and we do it in other places (verification), I'm okay with just leaving a TODO to address it later

**Review by jmrossy — APPROVED** [review]

File: packages/react-components/components/WithTextSearchPasteAware.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    paddingRight: 8,
+  },
+  searchIconContainer: {
+    marginLeft: 17,
+    marginRight: 13,
+  },
+  pasteIconContainer: {
+    // marginLeft: 17,
+    // marginRight: 40,
+  },
```
Comment (jmrossy) [inline packages/react-components/components/WithTextSearchPasteAware.tsx]: dead code

**Pull request merged**